nwbwidgets/image.py

```diff
@@ -167,3 +232,54 @@ def show_rbga_image(rgb_image: RGBImage, neurodata_vis_spec=None):
     plt.axis("off")
 
     return fig
+
+
+def get_frame_shape(external_path_file: PathType):
+    external_path_file = Path(external_path_file)
+    if external_path_file.suffix in ['.tif', '.tiff']:
+        assert HAVE_TIF, 'pip install tifffile'
+        tif = TiffFile(external_path_file)
+        page = tif.pages[0]
+        return page.shape
+    else:
+        assert HAVE_OPENCV, 'pip install opencv-python'
+        cap = cv2.VideoCapture(str(external_path_file))
+        success, frame = cap.read()
+        cap.release()
+        return frame.shape
+
+
+def get_frame_count(external_path_file: PathType):
+    external_path_file = Path(external_path_file)
+    if external_path_file.suffix in ['.tif', '.tiff']:
+        assert HAVE_TIF, 'pip install tifffile'
+        tif = TiffFile(external_path_file)
+        return len(tif.pages)
+    else:
```

Comment:
should be elif, and else should be an error message saying this type of external file is not yet supported by nwb widgets